app/labor/follow_checker.rb

```diff
@@ -8,7 +8,9 @@ def initialize(follower, followable_type, followable_id)
   end
 
   def cached_follow_check
-    RedisRailsCache.fetch("user-#{follower.id}-#{follower.updated_at}/is_following_#{followable_type}_#{followable_id}", expires_in: 20.hours) do
+    return false unless follower
+
+    RedisRailsCache.fetch("user-#{follower.id}-#{follower.updated_at&.rfc3339}/is_following_#{followable_type}_#{followable_id}", expires_in: 20.hours) do
```

Comment:
is there a chance for `updated_at` to be nil?

Reply:
_checks schema_
Nope!
_updates code_